buffalo/cmd/dev.go

```diff
@@ -92,7 +92,13 @@ func startWebpack(ctx context.Context) error {
 		}
 	}
 
-	cmd := exec.CommandContext(ctx, webpack.BinPath, "--watch")
+	if _, err := os.Stat(webpack.DevServerPath); err != nil {
```

Comment:
Currently this isn't backward compatible. However, if you change it to the following, it should work with current `v0.11.0` applications.

```go
	bin := webpack.DevServerPath
	if _, err := os.Stat(bin); err != nil {
		logrus.Warnf("%s is not installed, please install webpack-dev-server locally, trying to fallback to %s", bin, webpack.BinPath)

		bin = webpack.BinPath
		if _, err := os.Stat(bin); err != nil {
			return errors.Errorf("could not find either %s or %s", webpack.DevServerPath, webpack.BinPath)
		}
	}

	cmd := exec.CommandContext(ctx, bin)
	cmd.Stdin = os.Stdin
	cmd.Stderr = os.Stderr
	cmd.Stdout = os.Stdout
	return cmd.Run()
```

Reply:
Yea, you're right. I'll fix it!